api/types/image/opts.go

```diff
@@ -36,7 +37,23 @@ type PullOptions struct {
 }
 
 // PushOptions holds information to push images.
-type PushOptions PullOptions
+type PushOptions struct {
+	All          bool
```

Comment:
Yes! It's a bit weird though, because it's only there so that `client.ImagePush` can use it, but there is no actual separate form field for `all tags` on the API level... 🙈

Reply:
That would be a breaking change on the Go client API though.

Reply:
I think we already had some of those in this release, e.g. the `PrivilegeFunc` below was changed to now accept a context, so perhaps it's fine to group that in the same release 😄